### PR TITLE
Add phileos.madefor.cc

### PR DESCRIPTION
**Pull request by Ryan-Te**
Adds phileos.madefor.cc to the madefor.cc websites

### CODE REVIEW

**Comment by Ryan-Te** [comment]
adds PhileOS to domain list

File: dns/domains.py [hunk 1]
```diff
@@ -28,6 +28,7 @@ class Domain(DomainOptional):
     "cash": { "cname": "mcjack123.github.io" },
     "craftos-pc": { "cname": "admiring-shannon-be238c.netlify.app" },
     "metis": { "cname": "squiddev-cc.github.io" },
+    "phileos" : {"cname": "ryan-te.github.io/POS/"},
```
Comment (SquidDev) [inline dns/domains.py]: This should just be the domain, not the whole path.

**Review by SquidDev — CHANGES_REQUESTED** [review]

File: dns/domains.py (hunk 1 above)
Comment (Ryan-Te) [reply dns/domains.py]: ok fixed

Comment (Ryan-Te) [reply dns/domains.py]: https://github.com/Ryan-Te/madefor.cc/blob/patch-1/dns/domains.py

**Pull request merged**